pkg/event/handler.go

```diff
@@ -47,8 +47,8 @@ type Handler interface {
 	// RemoteEndpointRemoved is called when an endpoint associated with a remote cluster is removed
 	RemoteEndpointRemoved(endpoint *submV1.Endpoint) error
 
-	// NodeAdded indicates when a node has been added to the cluster
-	NodeAdded(node *k8sV1.Node) error
+	// NodeCreated indicates when a node has been added to the cluster
+	NodeCreated(node *k8sV1.Node) error
```

Comment:
I detected this being odd respect to the others on the interface, so I updated it. Now all of them are *Created..